shared/middleware/helpers/storage_id.rb

```diff
@@ -63,7 +63,12 @@ def storage_encrypt_app_id(storage_id, app_id)
 
 def storage_id(endpoint)
   return nil if endpoint == 'shared'
+  return -1 if endpoint == 'readonly'
```

Comment:
The goal here is to get a system that ships by the 27th. After that we're going to create the NoSQL back-ends and for that we're going to have each kind of table live in it's own actual no-sql table.